pdns/stubresolver.cc

```diff
@@ -18,59 +18,76 @@
 // stubDoResolve
 static vector<ComboAddress> s_stubresolvers;
 
-/** Parse /etc/resolv.conf and add the nameservers to the vector
+/*
+ * Returns true if no stubresolvers are configured, while emitting a warning about this
+ */
+bool noStubsDefined()
```

Comment:
how about `bool resolversDefined`? The things that are defined are not stubs, they are resolvers, and non-reverse logic is less confusing.